src/modeler/AutoRest.Swagger/JsonConverters/PathLevelParameterConverter.cs

```diff
@@ -64,8 +64,23 @@ public override bool CanConvert(System.Type objectType)
                 // Removing the common parameters to avoid serialization errors
                 jo.Remove("parameters");
             }
-            return JsonConvert.DeserializeObject<Dictionary<string, Operation>>(jo.ToString(),
-                GetSettings(serializer));
+
+            var result = new Dictionary<string, Operation>();
+
+            foreach (JProperty operation in jo.Children())
+            {
+                try
+                {
+                    result[operation.Name] = JsonConvert.DeserializeObject<Operation>(operation.Value.ToString(),
+                        GetSettings(serializer));
+                }
+                catch (JsonException)
+                {
+                    // Skip
```

Comment:
I'll make a change to check for the name and skip deserialization for extensions. I'll also use logger to report the error if it happens.